### PR TITLE
Make the cliquet documentation importable.

### PR DESCRIPTION
**Pull request by ametaireau**
It is now possible to import the docs from another project, for instance in the
Sphinx configuration file, using `cliquet_docs.copy_docs(section,
destination)`.

### CODE REVIEW

File: cliquet_docs [hunk 1]
```diff
@@ -0,0 +1 @@
+docs
```
Comment (Natim) [inline cliquet_docs]: ?

Comment (ametaireau) [inline cliquet_docs]: that's a symbolic link.

**Comment by leplatrem** [comment]
This conflicts with #410.

We have to take a decision :)

**Comment by ametaireau** [comment]
We decided to let `cliquet` be the canonical reference for now, and have a `cliquet-protocol` repository which will import from cliquet. We can revert this decision later if we detect this is a problem for implementers.

**Pull request merged**